source/agora/node/Registry.d

```diff
@@ -833,6 +842,7 @@ private struct ZoneData
             return Header.RCode.NameError;
 
         reply.answers ~= payload.toRR(question.qname);
+        reply.authorities ~= this.toRR(); // optional
```

Comment:
I think most servers don't include this, but reading the RFC, this looks correct.